[2025-05-25T05:24:21.600+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-25T05:24:23.033+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_file_to_psql.generate_file scheduled__2025-01-17T00:00:00+00:00 [queued]>
[2025-05-25T05:24:23.350+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_file_to_psql.generate_file scheduled__2025-01-17T00:00:00+00:00 [queued]>
[2025-05-25T05:24:23.361+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-05-25T05:24:23.972+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): generate_file> on 2025-01-17 00:00:00+00:00
[2025-05-25T05:24:24.415+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=604) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-25T05:24:24.524+0000] {standard_task_runner.py:72} INFO - Started process 672 to run task
[2025-05-25T05:24:24.580+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_file_to_psql', 'generate_file', 'scheduled__2025-01-17T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/data_quality.py', '--cfg-path', '/tmp/tmptany87xt']
[2025-05-25T05:24:24.740+0000] {standard_task_runner.py:105} INFO - Job 75: Subtask generate_file
[2025-05-25T05:24:26.436+0000] {task_command.py:467} INFO - Running <TaskInstance: load_file_to_psql.generate_file scheduled__2025-01-17T00:00:00+00:00 [running]> on host b4cf172ed0b9
[2025-05-25T05:24:29.384+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='https://t.me/Shust_DE' AIRFLOW_CTX_DAG_OWNER='@Shust_DE' AIRFLOW_CTX_DAG_ID='load_file_to_psql' AIRFLOW_CTX_TASK_ID='generate_file' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-17T00:00:00+00:00'
[2025-05-25T05:24:29.397+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-25T05:24:31.307+0000] {job.py:229} INFO - Heartbeat recovered after 12.90 seconds
[2025-05-25T05:24:31.945+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-25T05:24:32.136+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-25T05:24:32.142+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_file_to_psql, task_id=generate_file, run_id=scheduled__2025-01-17T00:00:00+00:00, execution_date=20250117T000000, start_date=20250525T052423, end_date=20250525T052432
[2025-05-25T05:24:32.980+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-25T05:24:33.217+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
